styles/index.css

```diff
@@ -0,0 +1,93 @@
+body {
+    display: grid;
+    grid-template: 1fr 1fr 1fr 1fr / 3fr .25fr 3fr;
+    background: url(images/rainbow_background.jpg) no-repeat center center fixed;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+    margin: 25px 25px 25px 25px;
+}
+
+header {
+    grid-row: 1 / span 1;
+    grid-column: 1/ span end; 
+    text-align: center;
+}
+
+.temperature {
+    background: rgb(255, 255, 255, 70%);
+    grid-row: 3/ span 1; 
+    grid-column: 1/ span 1;
+    padding-left: 15px;
+}
+
+.temperature button {
+    margin-right: 5pt;
+    font-size: 15pt;
+    font-weight: bolder;
+}
+
+#temp-number {
+    font-size: 25pt;
+    font-weight: bold;
+    margin-top: 10pt;
+}
+
+.sky {
+    background: rgb(255, 255, 255, 70%);
```

Comment:
Consider ways you might restructure your CSS to avoid rewriting the same custom color.